### DISCUSSION TITLE
RaspberryMatic Addon installation in VM auf Synology DSM723

### DISCUSSION THREAD

**Discussion by NicoK-86** [OpenCCU/OpenCCU#2879] [🇩🇪 Fragen&Antworten] [answered]
Hallo,

ich versuche die pivccu-modules-dkms kernel modules unter HomeAssistant Supervised in einer VM zu installieren, allerdings bekommen ich immer den Fehler "-sh: apt: not found" als zugriff habe ich den ssh Zugang über Port 22222 genommen mit root rechten, wie auf der Website von HA beschrieben. Für die VM habe ich das Image von der HomeAsssistant website benutzt.

Kann mir jemand weiterhelfen?

Danke schon mal.

**Comment by jens-maus** [comment]
> ich versuche die pivccu-modules-dkms kernel modules unter HomeAssistant Supervised in einer VM zu installieren, allerdings bekommen ich immer den Fehler "-sh: apt: not found" als zugriff habe ich den ssh Zugang über Port 22222 genommen mit root rechten, wie auf der Website von HA beschrieben. Für die VM habe ich das Image von der HomeAsssistant website benutzt.

Moment mal. Du scheinst hier einiges durcheinander zu bringen. Was für ein Host-OS hast du konkret unter dem du "HomeAssistant Supervised" installiert hast? Denn du schreibst "für die VM hab ich das Image von der HomeAssistant Website benutzt"?? Hast du jetzt HA Supervised oder hast du HomeAssistantOS installiert?!? Das ganze liest sich nämlich nach einem ziemlichen Durcheinander.

**Reply by NicoK-86** [reply]
Ich hab die OVA mit HA Supervised benutzt und wollte das RaspberryMatic Addon nutzen
<img width="620" alt="Bildschirmfoto 2024-10-21 um 14 36 01" src="https://github.com/user-attachments/assets/0fd60f2c-110a-4a6e-a5fa-47ca13471592">

**Reply by jens-maus** [reply, marked as answer]
Und da ist dein Trugschluss zu erkennen: Du hast doch kein "HA Supervised" installiert, sondern eben das "HomeAssistant OS" das den Supervisor automatisch mitbringt. "HA Supervised" Installation bedeutet du hast z.B. ein Debian/Ubuntu Linux und installierst darin dann HomeAssistant nicht als VM sondern direkt innerhalb dieses OS. Das hast du aber nciht, denn du hast ja direkt HomeAssistantOS installiert. Und weil du das "OVA" installiert hast gehe ich davon aus du hast es unter Proxmox oder vmWare installiert und damit musst du dann auch keine pivccu-modules-dkms installieren. Befolge einfach die Anleitungsschritte für RaspberryMatic unter HomeAssistantOS: https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant#using-homeassistant-os

**Reply by jens-maus** [reply]
... und wenn du ohnehin HomeAssistant als VM laufen lässt würde ich dir auch vorschlagen RaspberryMatic ebenso als VM/OVA unter der Synology laufen zu lassen (siehe https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Synology-VMM) und dann einfach die "Homematic (IP) Local" Integration in HA dann darauf zu konfigurieren (siehe https://github.com/jens-maus/RaspberryMatic/wiki/HomeAssistant-Integration). Das wäre dann die "Best Practice" Variante.

**Comment by NicoK-86** [comment]
alles klar, vielen dank :-)
